Makefile

```diff
@@ -142,20 +146,14 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
-.PHONY: yq
-yq:
-	test -f yq || wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O yq
-	chmod +x yq
-
 # Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
-bundle: operator-sdk manifests yq
+bundle: operator-sdk manifests csv-generator
 	./operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | ./operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
-	./yq w -i bundle/manifests/ssp-operator.clusterserviceversion.yaml spec.webhookdefinitions[0].containerPort 9443
-	./yq d -i bundle/manifests/ssp-operator.clusterserviceversion.yaml spec.install.spec.deployments[0].spec.template.spec.volumes
-	./yq d -i bundle/manifests/ssp-operator.clusterserviceversion.yaml spec.install.spec.deployments[0].spec.template.spec.containers[0].volumeMounts
+	./csv-generator --csv-version $(VERSION) --namespace kubevirt --operator-image $(IMG) --file bundle/manifests/ssp-operator.clusterserviceversion.yaml --webhook-port 9443 --webhook-remove-certs > bundle/manifests/ssp-operator.clusterserviceversion.yaml.new
```

Comment:
I know that the `VERSION` var exist here, but we actually use `IMG_TAG` in the travis job (where it counts in this context), so lets use `IMG_TAG` here as well and decide if we want to use `VERSION` or remove it in a different PR

Reply:
Also, `--operator-version` is missing here, so the `OPERATOR_VERSION` env var comes out empty in the deployment defined in the CSV

Reply:
Thanks, I'll replace `VERSION` with `IMG_TAG`.

As for OPERATOR_VERSION, it comes out empty without this PR as well. I went for 0 diff between `make bundle` before and after this PR.
I was planning on fixing that along with other things as part of a separate effort, but I can do it here if preferred.

Reply:
Yes, I would prefer to take care of `OPERATOR_VERSION` as well, as it is a very low hanging fruit in this context

Reply:
While adding OPERATOR_VERSION, I noticed I forgot to replace VERSION with IMG_TAG for `--csv-version` above.
However, when I do that, csv-generator fails because IMG_TAG defaults to "latest" and csv-generator expects X.Y.Z.
Should I keep using VERSION, or fix csv-generator, or change the default value?
Thanks

Reply:
I think it would be safe to do the following changes:
```
IMG_TAG ?= v$(VERSION)
```
This way, there's no danger of overriding the `v0.0.1` image because we have no releases prior to `v0.1.0` WDYT?

Reply:
Switched to setting VERSION in travis-ci as discussed offline, force-pushed, let me know if that makes sense.